javagen/src/main/java/com/azure/autorest/template/SwaggerReadmeTemplate.java

```diff
@@ -78,6 +79,25 @@ public String write(Project project) {
         return builder.toString();
     }
 
+    private static void addRequireOrInputFile(Map<String, Object> objectNode, AutorestSettings autorestSettings) {
+        // try use "require"
+        boolean useRequire = false;
+        List<String> requireList = autorestSettings.getRequire();
+        if (!CoreUtils.isNullOrEmpty(requireList)) {
+            String require = requireList.iterator().next();
+
+            if (require.contains("data-plane")) {
```

Comment:
this is a hack for the usual location of data-plane readme e.g. https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/readme.md